docs/specs/event.md

```diff
@@ -145,6 +146,24 @@ Occurs right before the end-user schedules account deletion.
 }
 ```
 
+#### user.session.jwt.pre_create
+
+Occurs right before the access token is issued.
+Use this event to add custom fields to the JWT access token.
+
+```json5
+{
+  "payload": {
+    "user": { /* ... */ },
+    "payload": {
```

Comment:
`payload` in `payload` looks a bit strange.

How about using `jwt`, or `payload` inside `jwt`?